README.md

```diff
@@ -0,0 +1,129 @@
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ... 
+
+generic_mon_connecting_socket:
+    - type: interface
+    - description: Authorize to connect to socket.
+    - parameters: The domain subname to be authorized to connect to socket.
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_commom_access
+    - type: interface
+    - description: Authorize to get some basic infos for monitoring tools (processes, fs info).
+    - parameters: 
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_pid_file_trans
+    - type: interface
+    - description: Ensure pid files are labelled correctly.
+    - parameters: 
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_sudo_exec   
+    - type: interface
+    - description: Run commands through sudo.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_kernel_message    
+    - type: interface
+    - description: Read and send kernel messages.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_manage_tmp_files
+    - type: interface
+    - description: Manage tmp files and ensure they are labelled correctly.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_dmesg
+    - type: interface
+    - description: Allow to run dmesg command.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_systat
+    - type: interface
+    - description: Allow to run systat commands.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_write_devlog
+    - type: interface
+    - description: Allow to write to devlog sock file.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_service_status
+    - type: interface
+    - description: Allow to check service / unit status.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_read_dir_auditd_log
+    - type: interface
+    - description: Allow to read auditd log directories.
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+generic_mon_monitoring_metrics
+    - type: interface
+    - description:  Allow to bind on monitoring port
+    - parameters:
+        * <domain subname> i.e. the name of the monitoring application / purpose, ...
+
+## Disclaimer
+
+The code of the this SELinux policy module is provided AS-IS. People and organisation
+willing to use it must be fulling aware that they are doing it at their own risks and
```

Comment:
Fully

Reply:
Remove the capabilities from common interfaces.
Also create new interfaces for capabilities execmem and net_admin if needed by end user.